core/themes/basis/theme-settings.php

```diff
@@ -70,3 +70,95 @@
     $form['footer'][$field] = color_get_color_element($form['theme']['#value'], $field, $form);
   }
 }
+
+/**
+ * Altering the theme settings for CSS updates.
+ */
+function basis_form_system_theme_settings_alter(&$form, &$form_state, $form_id = NULL) {
+  // Ensure template.php is included.
+  $theme_name = $form['theme']['#value'];
+  $theme_path = backdrop_get_path('theme', 'basis');
+  require_once $theme_path . '/template.php';
+  $install_version = config_get('system.core', 'install_version');
+
+  $css_update_options = array(
+    'no_updates' => t('Default'),
+    'all_updates' => t('All updates'),
+    'custom' => t('Specific version.'),
+  );
+
+  $form['css_updates'] = array(
+    '#type' => 'details',
+    '#open' => FALSE,
+    '#summary' => t('CSS Updates'),
+    '#details' => t('There are occasionally updates available for Basis that may affect existing sites, changing this setting may cause unexpected changes to your theme. For more information and details, see the <a href="https://docs.backdropcms.org/documentation/layouts-and-templates" target="_blank">online documentation</a>.</br></br>Select which updates you would like to accept. '),
+    '#attributes' => array(
+      'class' => array('description'),
+    ),
+  );
+
+  // Load the saved configuration value.
+  $form['css_updates']['css_update'] = array(
+    '#title' => t('CSS Update Options'),
+    '#type' => 'radios',
+    '#options' => $css_update_options,
+    '#default_value' => theme_get_setting('css_update', $theme_name),
+    '#config' => 'basis.settings',
+    'no_updates' => array(
+      '#description' => t('Default: Accept no updates introduced after installation.'),
```

Comment:
Default is the label, no need to duplicate it in the description.
```suggestion
      '#description' => t('Accept no updates introduced after installation.'),
```

Reply:
Good point.